### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

File: docs/player-support/snapcast.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+Whether you're using Snapcast-compatible speakers or devices like the Raspberry Pi, you can enjoy synchronized audio playback effortlessly. This component is contributed and maintained by [SantiagoSotoC](https://github.com/Santiagosotoc)
 
-In order to use this provider you need to also have a Snapcast server running on your network. The diagram below shows a possible combination of outputs. In the diagram a Raspberry Pi runs the server which communicates to MA and all of the clients. The server running Pi is also running Snapclient and is connected to a set of speakers. Then there is another Pi running Snapclient in another room, a phone running Snapdroid and a laptop running Snapweb,
+In order to use this provider you need to also have a Snapcast server running on your network (a server is included with this provider). The diagram below shows a possible combination of outputs. In the diagram a Raspberry Pi runs the server which communicates to MA and all of the clients. The server running Pi is also running Snapclient and is connected to a set of speakers. Then there is another Pi running Snapclient in another room, a phone running Snapdroid and a laptop running Snapweb,
```
Comment (marcelveldt) [inline docs/player-support/snapcast.md:6]: I would rephrase this as I expect most people will just use the builtin server. So rewrite it to the fact that music assistant includes a snapserver but if you already have an existing snapserver running on your network, there's an option to use that one instead.

File: docs/player-support/snapcast.md [hunk 2]
```diff
@@ -14,18 +14,19 @@ In order to use this provider you need to also have a Snapcast server running on
 
 ## Known Issues / Notes
 
-- The Snapcast provider is not enabled by default. When enabled the Snapcast server IP and port must be entered. From this point the workings of the server are transparent and the clients appear in the MA UI. 
-- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions.
-- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings.
+- The Snapcast provider is not enabled by default. An inbuilt Snapserver is launched when this provider is enabled. Snapweb is enabled on this server. Once enabled the workings of the server are transparent and the clients appear in the MA UI
+- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions
```
Comment (marcelveldt) [inline docs/player-support/snapcast.md:18]:
```suggestion
- Music Assistant only supports Snapcast version 0.27.0 (or newer);
```

File: docs/player-support/snapcast.md [hunk 3]
```diff
@@ -14,18 +14,19 @@ In order to use this provider you need to also have a Snapcast server running on
 
 ## Known Issues / Notes
 
-- The Snapcast provider is not enabled by default. When enabled the Snapcast server IP and port must be entered. From this point the workings of the server are transparent and the clients appear in the MA UI. 
-- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions.
-- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings.
+- The Snapcast provider is not enabled by default. An inbuilt Snapserver is launched when this provider is enabled. Snapweb is enabled on this server. Once enabled the workings of the server are transparent and the clients appear in the MA UI
```
Comment (marcelveldt) [inline docs/player-support/snapcast.md:17]: I would rephrase this a bit

File: docs/player-support/snapcast.md [hunk 4]
```diff
@@ -14,18 +14,19 @@ In order to use this provider you need to also have a Snapcast server running on
...
-- Ensure the server is launched with the command "snapserver -v".
-- Ensure that the ports on the Snapserver host 1704, 1705 are open. Also ensure that for each client a port equal to or greater than 4953 is open.
+- The Snapcast app for iOS is broken as it uses an old version of Snapclient. Using it brings problems with this provider
+- If using an external server ensure it is launched with the command "snapserver -v".
+- Ensure that the ports 1704 and 1705 on the Snapserver host are open. Also ensure that for each client a port equal to or greater than 4953 is open
 - Try the default Snapcast settings and then make changes as necessary.
 - Client names for all clients can be adjusted in Snapweb and Snapdroid via their respective UIs
 
-## Install Snapserver
+## Install External Snapserver
```
Comment (marcelveldt) [inline docs/player-support/snapcast.md:29]: scrap this completely - users that want to run their own snapserver already know this and they just toggle the option to use an existing snapserver instead of the builtin one.

File: docs/player-support/snapcast.md [hunk 5]
```diff
@@ -14,18 +14,19 @@ In order to use this provider you need to also have a Snapcast server running on
...
-- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions.
-- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings.
+- The Snapcast provider is not enabled by default. An inbuilt Snapserver is launched when this provider is enabled. Snapweb is enabled on this server. Once enabled the workings of the server are transparent and the clients appear in the MA UI
+- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions
+- The Snapcast provider will use the inbult Snapserver by default although a switch in the settings allows the use of an external server if desired. When using an external server the server IP and port must be entered. 
+- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings
```
[the remark this answers is not in the document]
Comment (OzGav) [reply docs/player-support/snapcast.md:20]: I assume this is not possible with his provider?

**Review by jozefKruszynski — APPROVED** [review]
